### PR TITLE
Move $gentoo_{keywords,use} to puppet::package::gentoo

### PR DESCRIPTION
**Pull request by tampakrap**

### CODE REVIEW

File: manifests/package/gentoo.pp [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
+    }
   } else {
-    $keywords = $puppet::agent::gentoo_keywords
     $package  = $puppet::params::agent_package
-    $use      = $puppet::agent::gentoo_use
+    if $use {
+      $real_use = $use
+    } else {
+      $real_use = 'minimal'
+    }
```
Comment (igalic) [inline manifests/package/gentoo.pp]: why not [selectors?](https://docs.puppetlabs.com/puppet/3.7/reference/lang_conditional.html#selectors)

**Comment by igalic** [comment]
i love the :package: emoji ;)

File: manifests/package/gentoo.pp (hunk 1 above)
Comment (tampakrap) [inline manifests/package/gentoo.pp]: Updated, review please

Comment (igalic) [inline manifests/package/gentoo.pp]: has a slightly better flow now

**Comment by danieldreier** [comment]
+1, looks reasonable to me. Thanks for helping clean up this part of the codebase. @actown do you want to take a look, as a gentoo user?

**Comment by tampakrap** [comment]
On hold, I need to refactor it against #171 when it will be merged

**Comment by tampakrap** [comment]
I mean #177 not 171

File: manifests/package/gentoo.pp [hunk 2]
```diff
@@ -1,17 +1,22 @@
...
 
-  include puppet::agent
   include puppet::params
 
   if $puppet::server::master {
-    include puppet::server
-    $keywords = $puppet::server::gentoo_keywords
     $package  = $puppet::params::master_package
-    $use      = $puppet::server::gentoo_use
+    $real_use = $use ? {
```
Comment (actown) [inline manifests/package/gentoo.pp]: Why not just make this a default on line 3?

**Comment by actown** [comment]
+1

**Comment by tampakrap** [comment]
moved to #181

**Pull request closed without merging**